### PR TITLE
Check rotation of journal logs

### PR DESCRIPTION
**Pull request by mloviska**
Validate whether log rotation has been successful, fix minor typo and
upload complete content of journald log directory.

- Verification runs: 
   * [sle-15-SP3-JeOS-for-VMware-x86_64-Build24.50-jeos-extratest@svirt-vmware65](http://kepler.suse.cz/tests/4037#step/journalctl/200)
   *  [tw](http://kepler.suse.cz/tests/4038#step/journalctl/373)
   * [leap15.3](http://kepler.suse.cz/tests/4032#step/journalctl/355)

### CODE REVIEW

File: tests/console/journalctl.pm [hunk 1]
```diff
@@ -61,8 +66,11 @@ sub verify_journal {
     } elsif (defined($fss_key) && (script_run("egrep 'No sealing yet,.*of entries not sealed.' errs") == 0)) {
         die "Sealing is not working!\n";
         # Check for https://bugzilla.suse.com/show_bug.cgi?id=1178193, a race condition for `journalctl --verify`
-    } elsif ((script_run("grep 'File corruption detected' errs") == 0) && script_run("$cmd")) {
+        #    } elsif ((script_run("grep 'File corruption detected' errs") == 0)) {
```
Comment (jlausuch) [inline tests/console/journalctl.pm:69]: left over?

File: tests/console/journalctl.pm [hunk 2]
```diff
@@ -51,7 +56,7 @@ sub verify_journal {
     my $cmd = 'journalctl --verify';
     $cmd .= " --verify-key=$fss_key" if defined($fss_key);
 
-    script_run('journactl --flush');    # ensure data is written to persistent log
+    script_run('journalctl --flush');    # ensure data is written to persistent log
```
Comment (jlausuch) [inline tests/console/journalctl.pm:59]: I wonder what have been checking so far with such a fail....

File: tests/console/journalctl.pm (hunk 1 above)
Comment (mloviska) [reply tests/console/journalctl.pm]: yup, haven't pushed the latest change. thanks for letting me know

File: tests/console/journalctl.pm (hunk 2 above)
Comment (mloviska) [reply tests/console/journalctl.pm]: It should have ensured that we have all previous entries written to the file. So not a check of itself. More of prior preparation for following ones.

**Review by grisu48 — APPROVED** [review]
Good work, LGTM! :rocket:
It looks to me like this is catching the remaining corner cases, right?

**Review by mloviska — COMMENTED** [review]
> Good work, LGTM! rocket
> It looks to me like this is catching the remaining corner cases, right?

Yup, and I have few ideas thanks to Franck when we were debugging the vmware issue with boot logs. And ofc, he explained me few perks of journald. Thankful for his insight

**Review by jlausuch — APPROVED** [review]

**Comment by michaelgrifalconi** [comment]
Hello, sorry to jump in. This PR seems to have broken the following tests:
- https://openqa.suse.de/tests/5688730
- https://openqa.suse.de/tests/5688854
- https://openqa.suse.de/tests/5688784

Therefore we are going to revert this and check if this resolves the problem.

**Comment by mloviska** [comment]
> Hello, sorry to jump in. This PR seems to have broken the following tests:
> 
>     * https://openqa.suse.de/tests/5688730
> 
>     * https://openqa.suse.de/tests/5688854
> 
>     * https://openqa.suse.de/tests/5688784
> 
> 
> Therefore we are going to revert this and check if this resolves the problem.

Thanks, let me check

**Comment by mloviska** [comment]
> Hello, sorry to jump in. This PR seems to have broken the following tests:
> 
>     * https://openqa.suse.de/tests/5688730
> 
>     * https://openqa.suse.de/tests/5688854
> 
>     * https://openqa.suse.de/tests/5688784
> 
> 
> Therefore we are going to revert this and check if this resolves the problem.

Actually it looks like a bug, as Birth field is empty. 

```
#   File: '/var/log/journal/fd5039c872a7eab7521cfb646052a165/system.journal'
  Size: 8388608   	Blocks: 16384      IO Block: 4096   regular file
Device: 32h/50d	Inode: 314         Links: 1
Access: (0640/-rw-r-----)  Uid: (    0/    root)   Gid: (  493/systemd-journal)
Access: 2021-03-18 07:16:15.840980540 -0400
Modify: 2021-03-18 07:54:49.092980540 -0400
Change: 2021-03-18 07:54:49.092980540 -0400
 Birth: -
 ```

**Comment by mloviska** [comment]
> Hello, sorry to jump in. This PR seems to have broken the following tests:
> 
>     * https://openqa.suse.de/tests/5688730
> 
>     * https://openqa.suse.de/tests/5688854
> 
>     * https://openqa.suse.de/tests/5688784
> 
> 
> Therefore we are going to revert this and check if this resolves the problem.

The issue is in package `coreutils-8.29-2.12.x86_64` and not file system related as the `brtime` is not present on xfs or btrfs based images/installations.
I have tried to update `basesystem` on sle15sp2 by the same module from sle15sp3 which has newer `coreutils-8.32-1.1.x86_64` package. 

```
# cat /etc/os-release
NAME="SLES"
VERSION="15-SP2"
VERSION_ID="15.2"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP2"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp2"
# stat /var/log/journal/3f983887cc46449199b909fde2407d2c/system.journal
  File: /var/log/journal/3f983887cc46449199b909fde2407d2c/system.journal
  Size: 8388608         Blocks: 16384      IO Block: 4096   regular file
Device: fd03h/64771d    Inode: 8388705     Links: 1
Access: (0640/-rw-r-----)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2021-03-18 13:31:48.578208323 +0000
Modify: 2021-03-18 13:31:48.838204334 +0000
Change: 2021-03-18 13:31:48.838204334 +0000
 Birth: -
 ```
 vs
 
 ```
 # stat /var/log/journal/3f983887cc46449199b909fde2407d2c/system.journal
  File: /var/log/journal/3f983887cc46449199b909fde2407d2c/system.journal
  Size: 8388608         Blocks: 16384      IO Block: 4096   regular file
Device: fd03h/64771d    Inode: 8388714     Links: 1
Access: (0640/-rw-r-----)  Uid: (    0/    root)   Gid: (  486/systemd-journal)
Access: 2021-03-18 13:58:15.537460176 +0000
Modify: 2021-03-18 13:58:25.837200513 +0000
Change: 2021-03-18 13:58:25.837200513 +0000
 Birth: 2021-03-18 13:58:15.533460277 +0000
 ```
 
 Not sure what to do, as `coreutils` depends on `glibc` @jlausuch wdyt ? Is it worth to report it as a bug or let's scope this check for currently in dev systems only?

**Pull request merged**